public/images/plans/plan-blogger.svg

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this should be minified too?

Reply:
It should! This is just a placeholder though, I only just received the final icon